.github/dependabot.yaml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - allow:
+      - dependency-type: direct
+    commit-message:
+      prefix: chore
+      include: scope
+    directory: '/'
+    open-pull-requests-limit: 99
+    package-ecosystem: npm
+    schedule:
+      interval: daily
+    versioning-strategy: increase
```

Comment:
Should we add updates of Actions too?

Reply:
@merlinnot What do you mean? Can you be more specific?

Reply:
Dependabot supports multiple package managers, e.g. npm and Docker. It also supports updates to Actions in all GitHub Workflows: https://github.blog/2020-06-25-dependabot-now-updates-your-actions-workflows/.